### PR TITLE
refactor covid_hosp - subdir for state timeseries

### PR DESCRIPTION
**Pull request by dfarrow0**
HHS now has 4 separate datasets under the "Reported Patient Impact and 
Hospital Capacity" umbrella, and this existing scraper is just for the 
single "State Timeseries" dataset.

This PR moves all the existing code for `covid_hosp` into a new subdir. Related scrapers will be added to sibling subdirs. Common code will be extracted back up into the original `covid_hosp` dir.

This PR should be a no-op operationally.

Note: at the ~same time as merge, automation step #73 needs to be updated from `delphi.epidata.acquisition.covid_hosp.update` to `delphi.epidata.acquisition.covid_hosp.state_timeseries.update`

### CODE REVIEW

**Comment by dfarrow0** [comment]
forgot to mention, all unit and integrations passed

File: src/acquisition/covid_hosp/state_timeseries/test_utils.py [hunk 1]
```diff
@@ -18,10 +18,17 @@
 
 class TestUtils:
 
-  def __init__(self, path_to_repo_root):
-    self.data_dir = (
-      Path(path_to_repo_root) / 'testdata/acquisition/covid_hosp/'
-    ).resolve()
+  def __init__(self, abs_path_to_caller):
+    # navigate to the root of the delphi-epidata repo
```
Comment (dfarrow0) [inline src/acquisition/covid_hosp/state_timeseries/test_utils.py:22]: no more `path.parent.parent.parent.parent` shenanigans

File: src/acquisition/covid_hosp/state_timeseries/test_utils.py [hunk 2]
```diff
@@ -18,10 +18,17 @@
...
+    path_to_repo = Path(abs_path_to_caller)
+    while path_to_repo.name != 'delphi-epidata':
+      if not path_to_repo.name:
+        raise Exception('unable to determine path to delphi-epidata repo')
+      path_to_repo = path_to_repo.parent
+
+    # path to the data, relative to the root of the repo
+    rel_data_path = 'testdata/acquisition/covid_hosp/state_timeseries/'
```
Comment (dfarrow0) [inline src/acquisition/covid_hosp/state_timeseries/test_utils.py:30]: will extract this into a constant in the next PR, which will make some bigger changes to this file

**Review by chinandrew — APPROVED** [review]
:shipit: :shipit: :shipit:

**Comment by dfarrow0** [comment]
deploy failed, i know why, working on fix

note: it actually deployed, and the code runs. unit test is broken because of fun with temporary paths

**Comment by dfarrow0** [comment]
should be fixed in https://github.com/cmu-delphi/delphi-epidata/pull/320 (fingers crossed)

**Pull request merged**